### PR TITLE
docs - text and sequencefile profiles support compression codec aliases

### PR DESCRIPTION
**Pull request by lisakowen**
doc changes for https://github.com/greenplum-db/pxf/pull/535.

in this PR:
- can use codec aliases for specifying COMPRESSION_CODEC in text and SequenceFile profiles.
- add statement upfront in parquet/text/sequencefile docs about the compression codecs supported.

question - can someone verify the compression codecs supported for sequencefile?  previous docs had bzip2 and gzip.

doc review site links:
- text - http://docs-lisa-pxf6-codecalias.cfapps.io/pxf/6-0/using/hdfs_text.html#hdfswrite_text
- sequencefile - http://docs-lisa-pxf6-codecalias.cfapps.io/pxf/6-0/using/hdfs_seqfile.html#hdfswrite_writeextdata

### CODE REVIEW

File: docs/content/hdfs_seqfile.html.md.erb [hunk 1]
```diff
@@ -56,20 +58,15 @@ The specific keywords and values used in the Greenplum Database [CREATE EXTERNAL
...
+When you use the `hdfs:SequenceFile` profile to write SequenceFile format data, you must provide the name of the Java class to use for serializing/deserializing the binary data. This class must provide read and write methods for each data type referenced in the data schema. You also specify the Java serialization class via a custom option to the `CREATE EXTERNAL TABLE` `LOCATION` clause.
 
 The `hdfs:SequenceFile` profile supports the following custom options:
 
 | Option  | Value Description |
 |-------|-------------------------------------|
-| COMPRESSION_CODEC    | The compression codec Java class name. If this option is not provided, Greenplum Database performs no data compression. Supported compression codecs include:<br>`org.apache.hadoop.io.compress.DefaultCodec`<br>`org.apache.hadoop.io.compress.BZip2Codec`<br>`org.apache.hadoop.io.compress.GzipCodec` |
+| COMPRESSION_CODEC    | The compression codec alias. Supported compression codecs include: `default`, `bzip2`, and `gzip`. If this option is not provided, Greenplum Database performs no data compression. |
```
Comment (ashuka24) [inline docs/content/hdfs_seqfile.html.md.erb:69]: If they explicitly want to say `uncompressed` that is also now an option here as well

File: docs/content/hdfs_parquet.html.md.erb [hunk 2]
```diff
@@ -23,6 +23,8 @@ under the License.
 
 Use the PXF HDFS connector to read and write Parquet-format data. This section describes how to read and write HDFS files that are stored in Parquet format, including how to create, query, and insert into external tables that reference files in the HDFS data store.
 
+PXF supports reading or writing Parquet files compressed with these codecs: `snappy`, `gzip`, and `lzo`.
```
Comment (bradfordb-vmware) [inline docs/content/hdfs_parquet.html.md.erb:26]: @denalex - Is there a reason that we support different codecs for Parquet vs SequenceFile/Text? Looking back at some old PRs (https://github.com/greenplum-db/pxf/pull/389) it looks like it was just a dependency issue?

**Review by bradfordb-vmware — APPROVED** [review]

**Review by mperezfuster — APPROVED** [review]

File: docs/content/hdfs_parquet.html.md.erb (hunk 2 above)
Comment (denalex) [reply docs/content/hdfs_parquet.html.md.erb:26]: I think this is mostly due to the set of codecs that given readers (parquet , orc, Avro) support

File: docs/content/hdfs_seqfile.html.md.erb [hunk 3]
```diff
@@ -56,20 +58,15 @@ The specific keywords and values used in the Greenplum Database [CREATE EXTERNAL
...
-
-- org.apache.hadoop.io.compress.DefaultCodec
-- org.apache.hadoop.io.compress.BZip2Codec
-
-When you use the `hdfs:SequenceFile` profile to write SequenceFile format data, you must provide the name of the Java class to use for serializing/deserializing the binary data. This class must provide read and write methods for each data type referenced in the data schema.
+SequenceFile format data can optionally employ record or block compression. You specify the compression type and codec via custom options to the `CREATE EXTERNAL TABLE` `LOCATION` clause.
 
-You specify the compression codec and Java serialization class via custom options in the `CREATE EXTERNAL TABLE` `LOCATION` clause.
+When you use the `hdfs:SequenceFile` profile to write SequenceFile format data, you must provide the name of the Java class to use for serializing/deserializing the binary data. This class must provide read and write methods for each data type referenced in the data schema. You also specify the Java serialization class via a custom option to the `CREATE EXTERNAL TABLE` `LOCATION` clause.
```
Comment (denalex) [inline docs/content/hdfs_seqfile.html.md.erb:63]: `... you must provide the name of Java .. for serializing/deserializing ... You also specify the Java serialization class ...` -- this sounds like we are talking about the same thing twice ? Maybe remove the word `also` if the second sentence is about HOW you do what needs to be done based on the first sentence ?

File: docs/content/hdfs_text.html.md.erb [hunk 4]
```diff
@@ -23,6 +23,8 @@ under the License.
 
 The PXF HDFS Connector supports plain delimited and comma-separated value form text data. This section describes how to use PXF to access HDFS text data, including how to create, query, and insert data into an external table that references files in the HDFS data store.
 
+PXF supports reading or writing text files compressed with the `bzip2` and `gzip` codecs.
```
Comment (denalex) [inline docs/content/hdfs_text.html.md.erb:26]: I think we also talk about `default` codec which we support..

**Review by denalex — APPROVED** [review]

**Review by ashuka24 — APPROVED** [review]

**Pull request merged**